sample/index.php

```diff
@@ -112,8 +113,16 @@ function isValidURL($url){
     </div>
 </div>
 
-<script src="js/jquery.min.js"></script>
+<script src="js/jquery-2.2.0.min.js"></script>
 <script src="js/bootstrap.min.js"></script>
-
+<script>
+$( document ).ready(function() {
+  $("#form-extractor").submit(function() {
```

Comment:
This actually prevents original "Extract" functionality to fail
Please address that as well